test/clj/puppetlabs/trapperkeeper/services/webserver/jetty9_service_test.clj

```diff
@@ -942,8 +942,10 @@
       app
       [jetty9-service
        hello-webservice]
-      (assoc-in jetty-ssl-pem-config [:webserver :ssl-protocols] ["SSLv3"])
-       (let [response (http-get "https://localhost:8081/hi_world" (merge default-options-for-https-client
-                                                                         {:ssl-protocols ["SSLv3"]}))]
-         (is (= (:status response) 200))
-         (is (= (:body response) "Hi World")))))))
+      (-> jetty-ssl-pem-config
```

Comment:
Shouldn't this test only verify that SSLv3 is permanently disabled? We should not allow SSLv3 to be used ever.

Reply:
I think this is just testing the configurability of the setting in general. It could also be tested by using the setting to disable TLSv1.1 as well. (or at least that's what I got from the PR/ticket: https://github.com/puppetlabs/trapperkeeper-webserver-jetty9/pull/92/files)

Reply:
Ahhh okay, sorry! Still getting the lay of the land with the other parts of the stack.